js/api.js

```diff
@@ -59,10 +59,11 @@ const baseAttrs = ['name', 'permalink', 'subreddit', 'score', 'num_comments', 't
 
 // array of post types, how to match, and how to display them
 const postTypes = [
-  { type: 'Video', match: /\.(webm|mp4)$/i },
+  { type: 'Video', match: /\.(webm|mp4)$/i, fields: ['author'] },
```

Comment:
this is totally fine, but since all the post types need this field it might be cleaner to add it to the `baseAttrs` array above.

Reply:
Fixed